build.gradle

```diff
@@ -29,6 +29,9 @@ sourceSets {
     java {
       srcDirs = ['test']
     }
+    resources {
+      srcDirs = ['testResources']
```

Comment:
Could we make it `test/resources` instead of `testResources`?  Just want to minimize the number of folders at the root.  If we need to (not sure how Gradle handles it), we can change `srcDirs = ['test/src']`.

Reply:
So we would move all test classes down one level into the new src directory?